csrc/scheduler/reduction_utils.h

```diff
@@ -139,5 +136,16 @@ std::string toString(ReductionType reduction_type);
 ReductionType getReductionType(Fusion* fusion);
 ReductionType getReductionType(const std::vector<TensorView*>& reduction_tvs);
 
+// Vectorization of smem consumers, they were created with cacheAfter() and
```

Comment:
Please mention what this function does and how it should be used. All mentioned here are some implementation limitations, but it's unclear what the expected result is by using this function.